CMakeLists.txt

```diff
@@ -166,8 +165,8 @@ endif()
 
 list(APPEND CMAKE_PREFIX_PATH "${CMAKE_BINARY_DIR}")
 
+include(CTest)
```

Comment:
Why include `CTest` if `BUILD_TESTING` is off?

Reply:
This is actually how it's recommended in the original `CTest` guidelines. The trick is that if you do that, then `BUILD_TESTING` by default is `ON`, not the other way around:
https://stackoverflow.com/questions/50468620/what-does-enable-testing-do-in-cmake

I looked at a few projects, most of them do about the same. I do not have a strong preference, but I think building tests by default is not a bad idea.

Reply:
Ok. Seems line 87 which defines `BUILD_TESTING` option could be deleted then as it is defined in `include(CTest)`.